UnityPy/classes/AnimationClip.py

```diff
@@ -101,7 +101,7 @@ def UnpackFloats(
                         bitPos = 0
 
                 x &= uint((1 << self.m_BitSize) - 1)  # (uint)(1 << m_BitSize) - 1u
-                data.append(x / (scale * ((1 << self.m_BitSize) - 1)) + self.m_Start)
+                data.append(x / (scale * ((1 << self.m_BitSize) - 1) if self.m_BitSize != 0 else 1 ) + self.m_Start)
```

Comment:
Same as above - float("inf").

Reply:
Infinity is a better choice. I'll fix it